.github/workflows/build.yml

```diff
@@ -413,6 +396,22 @@ jobs:
           path: |
             cudart-llama-bin-win-cu${{ matrix.cuda }}-x64.zip
 
+  freeBSD-latest:
+    runs-on: macos-12
```

Comment:
why is it run on macos?

Reply:
The documentation specify the latest supported macOS version is 12.
https://github.com/cross-platform-actions/action#runners

But it also support Linux, I guess we can try Ubuntu latest.
Do you think it worth change to linux?

Reply:
well, it should be fine, since this is an PUBLIC opensource project. If it where PRIVATE, we would have to pay 5x the compute minutes. :laughing: